helm/charts/determined/values.yaml

```diff
@@ -363,7 +363,7 @@ resourcePools:
 #   - resource_manager:
 #       type: kubernetes
 #       max_slots_per_pod: 1
-#       name: carolina-multirm-1
+#       cluster_name: carolina-multirm-1
```

Comment:
its not from this PR, but i think `carolina-multirm-1` shouldn't be in the main branch?

Reply:
Agreed! Changed this